customization-base/src/main/java/com/azure/autorest/customization/MethodCustomization.java

```diff
@@ -100,7 +104,22 @@ public MethodCustomization addAnnotation(String annotation) {
      * @return the current method customization for chaining
      */
     public MethodCustomization removeAnnotation(String annotation) {
-        return Utils.removeAnnotation(annotation, this, () -> refreshCustomization(methodSignature));
+        CompilationUnit compilationUnit = StaticJavaParser.parse(editor.getFileContent(fileName));
+        Optional<AnnotationExpr> annotationExpr = compilationUnit.getClassByName(className).get()
+            .getMethodsByName(methodName)
+            .stream()
+            .filter(method -> Utils.declarationContainsSymbol(method.getRange().get(), symbol.getLocation().getRange()))
+            .findFirst().get()
+            .getAnnotationByName(Utils.cleanAnnotationName(annotation));
+
+        if (annotationExpr.isPresent()) {
+            annotationExpr.get().remove();
+            editor.replaceFile(fileName, compilationUnit.toString());
+            Utils.sendFilesChangeNotification(languageClient, fileUri);
+            return refreshCustomization(methodSignature);
+        } else {
+            return this;
+        }
```

Comment:
This is very similar to the removeAnnotation method on the ConstructorCustomization class. We should use a utility method to reduce duplication.

Reply:
Yeah, I think we can add a utility method that takes an `Optional<AnnotationExpr>` and if present it does the ending logic.